### DISCUSSION TITLE
Not syncing albums with iOS

### DISCUSSION THREAD

**Discussion by kepler2214** [immich-app/immich#4143] [General]
Hello,

I just started experimenting with Immich and I have to say, this is by far the best self hosting photo system I have come across. It's really great!
I only encountered one thing which I'm unsure whether it's a bug or a missing feature. On my iPhone I have my images sorted in albums. If I sync these via the Immich app to the server, all images get transfered, but the albums do not (although the log on my phone says: 'Added a new local album to DB: example_album'). So I have to manually sort them back into albums on the server again.
Is this a known issue/bug or is it a feature that'll be implemented for future releases?

Thanks in advance!

**Comment by bo0tzz** [comment]
This isn't supported yet, see  #1678

**Reply by kepler2214** [reply]
Ah ok, haven't seen that. Thanks though!
